runtime/common/ArgumentConversion.cpp

```diff
@@ -199,8 +200,8 @@ Value dispatchSubtype(OpBuilder &builder, Type ty, void *p, ModuleOp substMod,
         return {};
       })
       .Case([&](cudaq::cc::CharspanType strTy) {
-        return genConstant(builder, *static_cast<const std::string *>(p),
-                           substMod);
+        return genConstant(
+            builder, (*static_cast<cudaq::pauli_word *>(p)).str(), substMod);
```

Comment:
```suggestion
            builder, static_cast<cudaq::pauli_word *>(p)->str(), substMod);
```
Does this work as well?

Reply:
Yes, updated. Thank!